src/main/java/Problem3/MovieAction.java

```diff
@@ -22,5 +24,14 @@ public int getLateFeeInDollar() {
     @Override
     public int calcLateFees(int numOfDaysPastDue) {
         // homework
+        if (numOfDaysPastDue <= 0){
+            return 0;
+        }
+        if(0 < numOfDaysPastDue && numOfDaysPastDue < 5){
+            return numOfDaysPastDue * 5;
+        }
+        else{
+            return 2 * (numOfDaysPastDue * 5);
```

Comment:
duplicated logic here. 

suggesting:
```java
        int fee = lateFeePerDayInDollar * numOfDaysPastDue;
        return (numOfDaysPastDue >= 5) ? 2 * fee : fee;
```

and here number 5 only appears once too